### PR TITLE
DIGI_33_UniqueTaxParameterTitle

### PR DESCRIPTION
**Pull request by prakashkolli**
1.added controller
2.added create View
3.added TaxParameter library

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    public class TaxParameterController : Controller
+    {
+        TaxParameterLibrary taxParameterLibrary = null;
+
+        /// Author: Prakash Kolli
+        /// Date: 4/08/2020
+        /// Description: Insert A TaxParameter TO Database
+        /// Ref. StoryId: DIGI 33
+        /// Reviewer:
+        /// Version: 1.0
```
Comment (bommavikas) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs:21]: No Need Of Summery In Controller

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                }   
+                else if(res.IsCompleted && res.Result.StatusCode ==HttpStatusCode.Created)
+                {
+                    TempData["fmessage"] = "Duplicate Values Are Not Allowed";
+                    return RedirectToAction("Create");
+                }
+            }
+            catch (Exception ex)
+            {
+                TempData["fmessage"] = "Failed To Insert Record";
```
Comment (bommavikas) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs:47]: Use Two Different TempData Varaibles For Duplicate message and Exception

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxParameterLibrary.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                httpResponse = libraryUtility.POSTAsync(EndPoint.NEW_TAXPARAMETER, jsonString);
+            }
+            catch (Exception ex)
+            {
+                throw ex;
+            }
+            return httpResponse;
+        }
+    }
+}
```
Comment (bommavikas) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxParameterLibrary.cs:29]: Revert This File As it is Available In Create Story

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs (hunk 1 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs]: removed summary

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs (hunk 2 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxParameterController.cs]: used different tempdata

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxParameterLibrary.cs (hunk 3 above)
Comment (prakashkolli) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxParameterLibrary.cs]: reverted this file

**Pull request merged**